benchmark/_http-benchmarkers.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+const child_process = require('child_process');
+
+// The port used by servers and wrk
+exports.PORT = process.env.PORT || 12346;
+
+function AutocannonBenchmarker() {
+  this.name = 'autocannon';
+  this.autocannon_exe = process.platform === 'win32'
+                      ? 'autocannon.cmd'
+                      : 'autocannon';
+}
+
+AutocannonBenchmarker.prototype.present = function() {
+  const result = child_process.spawnSync(this.autocannon_exe, ['-h']);
+  return !(result.error && result.error.code === 'ENOENT');
+};
+
+AutocannonBenchmarker.prototype.create = function(options) {
+  const args = ['-d', options.duration, '-c', options.connections, '-j', '-n',
+                `http://127.0.0.1:${options.port}${options.path}` ];
+  const child = child_process.spawn(this.autocannon_exe, args);
+  child.stderr.pipe(process.stderr);
```

Comment:
Maybe it would be better to do this in `.run`. That way we don't have any side effects in the benchmarker classes.